src/GovUk.Education.ExploreEducationStatistics.Publisher/Functions/PublisherFunction.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using GovUk.Education.ExploreEducationStatistics.Publisher.Model;
+using GovUk.Education.ExploreEducationStatistics.Publisher.Services.Interfaces;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+
+namespace GovUk.Education.ExploreEducationStatistics.Publisher.Functions
+{
+    public class PublisherFunction
```

Comment:
Would be good to get a comment at the top here to explain the function of this class

Reply:
I've removed it now and replaced it with one class per function. The functions might get a bit complicated plus it will be clearer to see which of the injected dependencies are used by which function.